crates/jni/src/jni.rs

```diff
@@ -21,12 +21,43 @@ use figma_import::{fetch_doc, ConvertRequest, ProxyConfig};
 use jni::objects::{JByteArray, JClass, JObject, JString, JValue, JValueGen};
 use jni::sys::{jboolean, jint, JNI_VERSION_1_6};
 use jni::{JNIEnv, JavaVM};
-use layout::{
-    add_style, add_style_measure, compute_node_layout, get_node_layout, remove_view, set_node_size,
-};
+use layout::LayoutManager;
 use lazy_static::lazy_static;
-use log::{error, info, warn, LevelFilter};
-use std::sync::{Mutex, MutexGuard};
+use log::{error, info, LevelFilter};
+use std::collections::HashMap;
+use std::sync::atomic::{AtomicI32, Ordering};
+use std::sync::{Arc, Mutex, MutexGuard};
+
+lazy_static! {
+    static ref JAVA_VM: Mutex<Option<JavaVM>> = Mutex::new(None);
+    static ref LAYOUT_MANAGERS: Mutex<HashMap<i32, Arc<Mutex<LayoutManager>>>> =
+        Mutex::new(HashMap::new());
+}
+static LAYOUT_MANAGER_ID: AtomicI32 = AtomicI32::new(0);
+
+fn javavm() -> MutexGuard<'static, Option<JavaVM>> {
+    JAVA_VM.lock().unwrap()
+}
+
+fn create_layout_manager() -> i32 {
+    let manager = Arc::new(Mutex::new(LayoutManager::new()));
+    let mut hash = LAYOUT_MANAGERS.lock().unwrap();
```

Comment:
I think we'll need to define a policy on `unwrap()` soon (also for HAR); in this file an un-unwrappable value probably should generate a java exception and return.

Reply:
Made https://github.com/google/automotive-design-compose/issues/670